servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/TestServiceStreaming.java

```diff
@@ -75,6 +75,7 @@ final class TestServiceStreaming implements StreamingHttpService {
     }
 
     @Override
+    @SuppressWarnings("PMD.ImplicitSwitchFallThrough")
```

Comment:
Is it complaining about no break in the `default` case?

Reply:
Yes, good eye. Removed suppression and added a `break`